### DISCUSSION TITLE
Star new build Disabled

### DISCUSSION THREAD

**Discussion by MonsterDKing** [codemagic-ci-cd/codemagic-docs#2092] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1.- Push the branch to deploy

### Expected results

1.- Generate new build 

### Actual results

The button start new build is disabled i check all reopsitorys settings in teams 

<img width="614" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/6463552/36e95d5a-79d5-481e-ac9b-e7ecd5065489">

It is not my first build, I have generated around 48 without any problem but today I tried to generate a new one and it does not let me generate a new build in any way.

**Comment by altmas5** [comment]
Same here, I am also unable to add new Apps.

**Comment by carsonskjerdal** [comment]
My Select Branch is endlessly loading with the text "Branches are loading". I have confirmed my github is still connected as well. I can trigger a build through a github push.

**Comment by ultraon** [comment]
Same issue, when I click on the button "Start new build" it becomes grey, and nothing happens.

**Comment by ammelto** [comment]
+1 same issue, for me it just sits on "loading yaml" forever.

**Comment by GoodAndBeautiful** [comment]
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/94643048/4a6e5f02-7bac-43ec-8f1c-970a099625eb)

Same

**Comment by appsowner** [comment]
Hello,

Same issue

any solution?....the connection with github not function?.

Thanks!

**Comment by icarusdust** [comment]
Hi, it should be alright now. Please check and confirm!

**Reply by icarusdust** [reply]
@MonsterDKing @altmas5 @ultraon @ammelto @towcar @appsowner @GoodAndBeautiful

**Reply by altmas5** [reply]
Solved now, thanks @icarusdust !

**Comment by gonzalogauto** [comment]
Same problem here

**Reply by icarusdust** [reply]
@gonzalogauto you might need to refresh the page before trying.

**Reply by gonzalogauto** [reply]
Thanks! @icarusdust , now the job is running 🚀

**Comment by appsowner** [comment]
Hello,

Running now!

Thanks you.!

**Comment by alsopgroup** [comment]
I'm having this issue, but via FlutterFlow.  My builds have been in queue for hours on end.

**Reply by icarusdust** [reply]
@alsopgroup as it is a different issue, you are always welcome to start new threads. 

There are no queues atm and everything is up and running. You can check a couple of things for you case:

1. Firstly, start a new build.
2. If your build is still in queue, check if you have other ongoing builds

Keep me posted in case of any further assistance is needed.

**Comment by helinanever** [comment, marked as answer]
Hi everyone, we had an infrastructure incident that lasted from 21:00 UTC to 00:10 UTC. During that time, some users experienced issues with loading branches or codemagic.yaml, starting builds, and adding new applications. The problem affected public repositories and repositories using SSH keys for authentication. OAuth repositories were not affected. We apologize for any inconvenience caused.

**Comment by Sush-HD** [comment]
@helinanever - Is codeMagic currently impacted by a similar incident? We are having the same issue on one of our repo.

**Reply by alsopgroup** [reply]
Same

**Comment by dipak-onehop** [comment]
We are also facing the same issue. its not allowing to start a new build for our private repo as well. Any solutions?

**Comment by oudjet** [comment]
Hello, has the problem come back? 

<img width="608" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/129376744/c28d8e48-c847-45af-9f14-3d6c91d32b3d">

**Reply by ryoganaruse** [reply]
I am facing the same problem.

It occurs on November 16, 2023 at 24:00~November 17, 2023 at 12:57 (JST).
I pressed start new build in the same way, but the build is not working.
